src/pages/join/volunteering.js

```diff
@@ -30,12 +68,14 @@ TopicHeading.propTypes = {
 
 // -- MAIN
 export default () => {
+  const { theme } = useThemeContext();
 
   return (
     <Layout>
       <MainBlock>
         <TitleHeading>Volunteering</TitleHeading>
         <section>
+          <a href="https://airtable.com/shrlo3trJbLCvqPmA" target="_blank" rel="noreferrer">{btns[theme.toLowerCase()]}</a>
           <iframe title="volunteering-airtable" src="https://airtable.com/embed/shrFkKfMXPJBgbxtX?backgroundColor=green&amp;viewControls=on" frameborder="0" onmousewheel="" width="100%" height="533" style={{background: 'transparent', border: '1px solid #ccc'}}></iframe>
```

Comment:
maybe we can also add a title between the button the and iframe to make it more clear that the table is open/available positions